### PR TITLE
update requirements

### PR DESCRIPTION
**Pull request by creviera**
# Description

Bumps SDK to version 0.2.0. Just opening this PR even though we're not ready to use the latest SDK yet since I needed to update the client requirements in order to build the latest SDK wheels here: https://github.com/freedomofpress/securedrop-debian-packaging/pull/204 (due to our package build logic)

### CODE REVIEW

**Comment by emkll** [comment]
After merging https://github.com/freedomofpress/securedrop-debian-packaging/pull/204 and the new sdk wheels live on the pypi mirror https://pypi.securedrop.org/simple/securedrop-sdk/ which now unblocks work here.

We should update https://github.com/freedomofpress/securedrop-client/blob/update-to-sdk-0.2.0/requirements/build-requirements.txt#L12 prior to merging to ensure the new version of the wheel is included in the builds.

I just restarted CI after the pypi mirror was updated, there is what appears to be a valid test failure here.

**Comment by eloquence** [comment]
I would suggest deferring merge until we tag the next client release, just out of conservatism (we don't need the changes in the SDK yet for the release bundled as SDW 0.5.0).

**Comment by creviera** [comment]
since we'll want functional tests that test (1) we are getting seen data from the server, and (2) we are sending seen data to the server, i'll want to add that to this pr instead: https://github.com/freedomofpress/securedrop-client/pull/1165

that way i can do functional tests all at once utilizing the new `create_dev_data` functionality that adds seen data automatically

**Comment by creviera** [comment]
> This branch is out-of-date with the base branch

**Comment by creviera** [comment]
> This branch is out-of-date with the base branch

I change my mind. This repo is not busy enough to force this check.

**Comment by creviera** [comment]
> I change my mind. This repo is not busy enough to force this check.

The strict status check to enforce that branches are always up-to-date with the base branch before merge has been removed. Feels like a heavy weight has been lifted!

File: requirements/dev-mac-requirements.txt [hunk 1]
```diff
@@ -771,12 +383,8 @@ requests==2.22.0 \
...
     --hash=sha256:9cf5292fcd0f598c671cfc1e0d7d1a7f13bb8085e9a590f48c010551dc6c4b31 \
     # via -r requirements/requirements.in, securedrop-sdk
-rubicon-objc==0.3.1 \
-    --hash=sha256:4d74732edca4059e22cdaf04806c22f8535ef2de2bc16fe085c3748fd757b5a1 \
-    --hash=sha256:51d19a274f438e004e8a4c39b2e4067d287e79eff80bde582f6b00855e25b345 \
-    # via mouseinfo
-securedrop-sdk==0.1.1 \
-    --hash=sha256:138ce7a717db519c3c8d19b9475d7660fb7095d7608e8802723e682a7415e677 \
+securedrop-sdk==0.2.0 \
+    --hash=sha256:c4a343077e8c0a38914e17f6369b830f1e361f9d66699b20803c07b39472357f \
```
Comment (emkll) [inline requirements/dev-mac-requirements.txt:387]: :heavy_check_mark:

**Review by emkll — APPROVED** [review]
Thanks @creviera looks good to me. I'll update the mac requirements and submit another PR to resolve, but this is good to merge

**Review by emkll — APPROVED** [review]
Re-stamping after rebase fix to #1179

**Pull request merged**